src/components/Select/select.module.scss

```diff
@@ -15,25 +20,27 @@
     }
 
     .multi-select-pills {
-        position: absolute;
-        z-index: 1;
+        align-items: center;
+        display: flex;
+        flex-direction: row;
         left: $space-xs;
         right: $space-xxl;
+        pointer-events: none;
+        position: absolute;
         top: 6px;
-        display: flex;
-        flex-direction: row;
-        align-items: center;
+        z-index: 1;
```

Comment:
Let's use scss variables in this file?

Reply:
Will send a subsequent pr. Having issues with my fork and lfs (met my quota), need to create a new fork and clone.